app/views/users/show.html.erb

```diff
@@ -1 +1,32 @@
-<p>In this page the requested user (based on the received parameter) will be displayed.</p>
+<div id="users-show">
+  <div class="user-card">
+    <img src="<%= @user.photo %>" alt="User photo">
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <p> Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+
+  <div class="user-bio">
+    <h2 class="section-title">Bio</h2>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <ul class="user-posts">
+    <% @posts.each.with_index(1) do |post, index| %>
+      <li class="post">
+        <h2 class="post-title">
+          <%= post.title %>
+          <span class="post-id">#<%= index %></span>
+        </h2>
+        <p class="post-body"><%= post.text %></p>
```

Comment:
- [ ] Kindly make use of the `most_recent_posts` callback function you created in your user model to render the 3 most recent posts for a user.